clinica/iotools/converters/oasis3_to_bids/oasis3_to_bids_cli.py

```diff
@@ -23,8 +23,6 @@ def cli(
     from clinica.utils.check_dependency import check_dcm2niix
     from clinica.utils.stream import cprint
 
-    check_dcm2niix()
```

Comment:
Why was this unnecessary ?

Reply:
We don't use `dcm2niix` with this converter

Reply:
Indeed that's a good reason...